indra/sources/bel/pybel_processor.py

```diff
@@ -452,7 +452,10 @@ def _get_evidence(u_data, v_data, edge_data):
     annotations['bel'] = edge_to_bel(u_data, v_data, edge_data)
 
     text_location = annotations.pop('TextLocation', None)
-    if text_location is not None:
+    if text_location:
+        # Handle dictionary text_location like {'Abstract': True}
+        if isinstance(text_location, dict):
+            text_location = list(text_location.keys())[0]
```

Comment:
I suspect the switch to a dict is in part to allow multiple types of section info, with multiple key/value pairs--otherwise it could just be a string. If this is the case just taking the first key (which will be non-deterministic in general) might not give what we want.